src/features/search/utils/tests/useFilterCount.test.ts

```diff
@@ -85,6 +97,21 @@ describe('getFilterCount', () => {
         aroundRadius: 20,
       },
     }
-    expect(getFilterCount(aroundMeSelected)).toEqual(1)
+    expect(renderHook(() => useFilterCount(aroundMeSelected)).result.current).toEqual(1)
+  })
+})
+
+describe('useFilterCount under 18', () => {
+  const maxPriceUnder18 = 30
+  beforeAll(() => {
+    mockedUseMaxPrice.mockImplementation(() => maxPriceUnder18)
+    jest.clearAllMocks()
+  })
+
+  it('returns 0 when no filters even when the max price is not 300 (under 18)', () => {
```

Comment:
tu peux rajouter 1 test: "returns 1 when filter is below the max price for underage".

Reply:
bonne idée 👍
https://github.com/pass-culture/pass-culture-app-native/pull/1726/commits/31d3fb9cd486f7a5c5e5f5cde30b89c77a462b8d